NewSpring/ApollosProject
# [DEV-4151] Added Bible Translation

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR updates the devotional scripture component to bring in the copyright and hardcodes the version. The version has to be hardcoded for now until we get the API updated in core.

### How do I test this PR?
Run the sim and see that the translation once again appears on the devotional scripture tab. 

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
